core/api/tests/test_business_plans.py

```diff
@@ -529,63 +549,41 @@ def test_status_filter(self, user, business_plan, _setup_bp_record_list):
 
         response = self.client.get(
             self.url,
-            {"business_plan_id": business_plan.id, "status": "A"},
-        )
-        assert response.status_code == 200
-        assert len(response.json()["records"]) == 4
-
-        response = self.client.get(
-            self.url,
-            {"business_plan_id": business_plan.id, "status": "U"},
-        )
-        assert response.status_code == 200
-        assert len(response.json()["records"]) == 0
-
-    def test_search_filter(self, user, business_plan, _setup_bp_record_list):
-        self.client.force_authenticate(user=user)
-
-        response = self.client.get(
-            self.url,
-            {"business_plan_id": business_plan.id, "search": "Planu2"},
+            {
+                "year_start": business_plan.year_start,
+                "year_end": business_plan.year_end,
+                "status": "A",
+            },
         )
         assert response.status_code == 200
-        assert len(response.json()["records"]) == 1
-        assert response.json()["records"][0]["title"] == "Planu2"
-
-    def test_invalid_bp_id(self, user, _setup_bp_record_list):
-        self.client.force_authenticate(user=user)
-
-        response = self.client.get(
-            self.url,
-            {"business_plan_id": 999},
-        )
-        assert response.status_code == 400
-
-    def test_invalid_year(self, user, _setup_bp_record_list, agency):
-        self.client.force_authenticate(user=user)
+        assert len(response.json()) == 4
+        assert response.json()[0]["status"] == "A"
 
         response = self.client.get(
             self.url,
             {
-                "agency_id": agency.id,
-                "year_start": 99,
-                "year_end": 999,
+                "year_start": business_plan.year_start,
+                "year_end": business_plan.year_end,
+                "status": "U",
             },
         )
-        assert response.status_code == 400
+        assert response.status_code == 200
+        assert len(response.json()) == 0
 
-    def test_invalid_agency(self, user, _setup_bp_record_list):
```

Comment:
Why did you delete this test? we need county / agency / sector / etc filters for this endpoint too.